viz-lib/src/visualizations/chart/Editor/ChartTypeSelect.jsx

```diff
@@ -1,9 +1,9 @@
-import { map } from "lodash";
+import { filter, includes, map } from "lodash";
 import React, { useMemo } from "react";
```

Comment:
```suggestion
import React, { useMemo } from "react";
import PropTypes from "prop-types";
```

Reply:
Boy were you quicker than me to solve that, hahaha